### PR TITLE
ADD: is it my address check on wallet details screen (closes #829)

### LINKED ISSUE #829: Is it my address?
Create simple tool that allows me to paste an onchain address to verify if it belongs to one of my wallets.

### PR DESCRIPTION
**Pull request by Overtorment**
ugly but it works. accessible via wallet -> details. works globally for all wallets.

image

image

### CODE REVIEW

File: screen/send/isItMyAddress.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      backgroundColor: colors.background,
+    },
+    text: {
+      color: colors.foregroundColor,
+    },
+  });
+
+  const handleUpdateAddress = nextValue => setAddress(nextValue.trim());
+
+  const checkAddress = async () => {
```
Comment (limpbrains) [inline screen/send/isItMyAddress.js:40]: Why async?

File: screen/send/isItMyAddress.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    for (const w of wallets) {
+      if (w.weOwnAddress(cleanAddress)) {
+        _result.push(loc.formatString(loc.is_it_my_address.owns, { label: w.getLabel(), address: cleanAddress }));
+      }
+    }
+
+    setResult(_result.join('\n\n'));
+  };
+
+  const onBarScanned = async value => {
```
Comment (limpbrains) [inline screen/send/isItMyAddress.js:52]: Why async?

File: screen/send/isItMyAddress.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    setAddress(value);
+  };
+
+  const importScan = async () => {
```
Comment (limpbrains) [inline screen/send/isItMyAddress.js:56]: Why async?

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Review by marcosrdz — APPROVED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/n3g3pdumtqhny9k3hmjge803h8

[download apk](http://filestorage.bluewallet.io:1488/1607368411.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**